crates/contract/lib.rs

```diff
@@ -3,832 +3,55 @@
 use ink_lang as ink;
 
 #[ink::contract]
-mod erc20 {
-    use ink_storage::{
-        traits::SpreadAllocate,
-        Mapping,
-    };
-
-    /// A simple ERC-20 contract.
+pub mod flipper {
```

Comment:
What prompted you to change from `erc20` to `flipper` here?

Reply:
We always used `flipper` as the standard contract which initially shows up in Playground. I just changed it to `erc20` while trying to make playground work with rc8 for the talk.

Reply:
Ah gotcha